src/main/java/me/hypherionmc/sdlinklib/config/configobjects/BotSettingsConfig.java

```diff
@@ -61,6 +61,10 @@ public class BotSettingsConfig {
     @SpecComment("The type of the status displayed on the bot. Valid entries are: PLAYING, STREAMING, WATCHING, LISTENING")
     public ActivityType botStatusType = ActivityType.PLAYING;
 
+    @Path("botStatusStreamingURL")
+    @SpecComment("The URL that will be used when the \"botStatusType\" is set to \"STREAMING\", required to display as \"streaming\".")
+    public String botStatusStreamingURL = "https://twitch.tv/KuryKat";
```

Comment:
Please don't use your own Twitch URL, before we get accused of advertising again. Is there any generic one we can use?

Reply:
Understood, I'm sorry, I was just trying to use some placeholder hehe 😅

Reply:
Maybe I can use the twitch URL: https://www.twitch.tv/twitch

Reply:
Yeah, that would be fine

Reply:
Fixed in commit [eb413e39110b40f1e3f73a2804a95446390018e7](https://github.com/hypherionmc/sdlink-lib/pull/5/commits/eb413e39110b40f1e3f73a2804a95446390018e7)